### ISSUE TITLE
Remove eccc specific data from ckan converter.

### ISSUE THREAD

**Issue by ianwallen** [metadata101/iso19139.ca.HNAP#68]
The following files 

https://github.com/metadata101/iso19139.ca.HNAP/blob/3.10.x/src/main/plugin/iso19139.ca.HNAP/extract-ckan-dataset.xsl#L65-L70

contains the following

        ```
<!-- TODO: Fixed value to the code for ECCC, check for FGP -->
        <owner_org>49E2ADF4-AD7A-43EB-85C8-6433D37ED62C</owner_org>
        <creator>Environment and Climate Change Canada</creator>
        <subject>
          <value>economics_and_industry</value>
        </subject>
```

I believe this should be changed to customizable so that DFO can use different values.
I'm not sure on the best approach to do this.

**Comment by ianwallen** [comment]
@geothorne could you look into this or have someone from Geocat look into this.

If these are required then we may need to have this updated so that it gets these values from a configuration as we don't want every department to modify the xslt file directly.

**Comment by geothorne** [comment]
@ianwallen looking at this further... am I correct to assume that DFO uses this xsl to prepare for CKAN open.canada.ca?

**How/Where is this xsl script? I would like to see/understand how this works within your publishing workflow. **.... I am not even sure if it is being used, from the ECCC prospective we are not currently using this script since we are moving to the more generic ca.HNAP. 

If DFO is using this xsl is used as a standalone tool than  you (@ianwallen ) could do as us see fit to it.

I am not even sure if open.canada.ca profile even actually accepts these hard coded values, particularly the org uuid for ECCC that seems to be the most important rather than the actual org name.

Final observation, is this information not managed with GeoNetwork within the Admin console and exchanged through the info.xml MEF file? 

Chris

**Comment by ianwallen** [comment]
@geothorne 
We had done a proof of concept in the past where we pushed some data directly to open data.  We used the code the @josegar74 had created for this.  The way it seems to work is that is takes the hnap records and runs it through the ckan xsl file to create a format that is acceptable by open data.  I believe for our test, we may have hard coded the values to be DFO specific.

I believe this may have originally been created to allow FGP to publish to open data.  (Or maybe this was specific for ECCC to publish to open data). Since FGP only support geospatial data at the moment we still need to ability to publish to open data directly.  I believe ECCC may need the ability to publish to open data directly as well.   

I believe that we should add this item as a topic at out next meeting.  This fall into the general topic of how we expect to use this plugin to publish content to Open Data. (Maybe this will be an external unrelated process?  In which case this ckan file may be removed)
